examples/launcher/src/components/Dom.jsx

```diff
@@ -8,12 +8,15 @@ const _decorateKeypaths = (el, keypath = []) => {
   }
 };
 const _el2Text = el => {
-  let result = '<' + el.tagName.toLowerCase();
-  for (let i = 0; i < el.attrs.length; i++) {
-    const attr = el.attrs[i];
-    result += ' ' + attr.name + '=' + JSON.stringify(attr.value);
+  let result = '';
```

Comment:
Isn't this deleting the old functionality, gutting the DOM view entirely?

Reply:
Yes, I'll revert that since DOM view is going to be used later on.

Reply:
aeafc1f

Reply:
Looks good 👍